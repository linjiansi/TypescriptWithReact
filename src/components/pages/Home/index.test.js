import React from 'react';
import renderer from 'react-test-renderer';
import { Home } from './index';

it('renders correctly at Home', () => {
  const tree = renderer
    .create(<Home></Home>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
