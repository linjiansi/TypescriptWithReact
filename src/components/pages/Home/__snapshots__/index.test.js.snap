// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly at Home 1`] = `
Array [
  <div
    className="makeStyles-root-2"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-header-4 MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <h6
          className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
        >
          BookManager
        </h6>
        <a
          href="/login"
          onClick={[Function]}
        >
          <div
            className="makeStyles-main-6"
          >
            <button
              className="MuiButtonBase-root-35 MuiButton-root-8 MuiButton-contained-16 makeStyles-button-7 MuiButton-containedSecondary-18"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label-9"
              >
                LOGOUT
              </span>
            </button>
          </div>
        </a>
      </div>
    </header>
  </div>,
  <div
    className="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation8 MuiPaper-rounded"
  >
    <div
      className="makeStyles-root-2"
    >
      <header
        className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorSecondary makeStyles-header-4 MuiPaper-elevation4"
      >
        <div
          className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
        >
          <h6
            className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
          >
            書籍一覧
          </h6>
          <a
            href="/add"
            onClick={[Function]}
          >
            <div
              className="makeStyles-main-6"
            >
              <button
                className="MuiButtonBase-root-63 MuiButton-root-36 MuiButton-contained-44 makeStyles-button-7 MuiButton-containedSecondary-46"
                disabled={false}
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                type="button"
              >
                <span
                  className="MuiButton-label-37"
                >
                  CREATE
                </span>
              </button>
            </div>
          </a>
        </div>
      </header>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <a
          href="/edit"
          onClick={[Function]}
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-64 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root makeStyles-media-65"
              style={
                Object {
                  "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
                }
              }
              title="book"
            />
            <div
              className="MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
              >
                書籍名: 
                hogehoge
              </p>
              <div
                className="sc-bdnxRM ctBBVd"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  金額: 
                  0
                </p>
                <p
                  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  購入日: 
                  
                </p>
              </div>
            </div>
          </div>
        </a>
      </div>
    </div>
  </div>,
]
`;
